src/app.js

```diff
@@ -0,0 +1,65 @@
+import axios
+    from "axios";
+
+const list = document.getElementById("country-list");
+
+async function getCountryData() {
+    try{
+// aanroepen van de data
+        const response = await axios.get("https://restcountries.com/v2/all");
+        const countries = response.data;
+// op volgorde zetten
+        countries.sort((a, b) => {
+            return a.population - b.population;
+        });
+
+// Met map functie over alle items in de lijst gaan.
+        countries.map((country) => {
```

Comment:
Je zou hier nog spreading kunnen gebruiken, zodat je niet overal `country.` voor hoeft te zetten
```suggestion
        countries.map({ name, flag, region, population } => {
```